modules/rostests/kmtests/ntos_mm/MmReservedMapping.c

```diff
@@ -281,6 +299,9 @@ START_TEST(MmReservedMapping)
     PVOID Mapping;
 
     g_IsPae = ExIsProcessorFeaturePresent(PF_PAE_ENABLED);
+    g_OsVersion = SharedUserData->NtMajorVersion << 8 | SharedUserData->NtMinorVersion;
+    g_IsReactOS = *(PULONG)(KI_USER_SHARED_DATA + PAGE_SIZE - sizeof(ULONG)) == 0x8eac705;
+    ok(g_IsReactOS == 1, "Not reactos\n");
```

Comment:
It would be of interest to have these global variables being global on the whole kmtests framework and initialized by the main() function of said kmtest.exe (and the .sys driver), so that we don't have to redo these same init everytime in kmtest.

Reply:
It should probably be done in a separate PR, not here I guess.

Reply:
I would like to leave this to a more knowledgeable person and another PR. Thanks.